fbfmaproom/fbflayout.py

```diff
@@ -335,18 +358,18 @@ def command_layout():
 
         ],
         id="command_panel",
-        className="info",
-        style={
-            "position": "absolute",
-            "top": "10px",
-            "right": "10px",
-            "left": "230px",
-            "zIndex": "1000",
-            "height": "fit-content",
-            "pointerEvents": "auto",
-            "paddingLeft": "10px",
-            "paddingRight": "10px",
-        },
+        # className="info",
+        # style={
+        #     "position": "absolute",
+        #     "top": "10px",
+        #     "right": "10px",
+        #     "left": "230px",
+        #     "zIndex": "1000",
+        #     "height": "fit-content",
+        #     "pointerEvents": "auto",
+        #     "paddingLeft": "10px",
+        #     "paddingRight": "10px",
+        # },
```

Comment:
Font has changed, text no longer fits in menus.

Reply:
The fonts in the control bar look good now, but the fonts in the table controls are still weird.